synapse/lib/stormlib/graph.py

```diff
@@ -0,0 +1,162 @@
+import synapse.exc as s_exc
+import synapse.lib.config as s_config
+import synapse.lib.stormtypes as s_stormtypes
+
+gdefSchema = {
+    'type': 'object',
+    'properties': {
+        'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'name': {'type': 'string'},
+        'scope': {'type': 'string', 'enum': ['user', 'global', 'power-up']},
+        'creator': {'type': 'string'},
```

Comment:
I think in similar cases elsewhere we usually make this the user iden instead of name, but that would break using the package name as the creator for power-up graphs?

Reply:
n/m, I see you just left it loose to account for both cases.   I wonder if they should be separated out since it feels a little overloaded.

Reply:
Yeah I couldn't decide between splitting it and just tossing them both in creator. Probably better to split it up though since they will likely need to be handled differently anyways.